app/assets/javascripts/discourse/app/components/topic-timeline/back-button.hbs

```diff
@@ -2,7 +2,7 @@
   @type="button"
   @class="btn-primary btn-small back-button"
   @title={{i18n "topic.timeline.back_description"}}
-  @onClick={{@onClick}}
+  @action={{@goBack}}
 >
   {{i18n "topic.timeline.back"}}
-</DButton>
\ No newline at end of file
+</DButton>
```

Comment:
Why does GH UI show there are changes on the last lines of those hbs files? 🤨

Reply:
@janzenisaac you likely need to update your editor config to stop it adding newlines to the end of `.hbs` files - it's a prettier violation. (which was previously being ignored by GitHub actions - fixed by https://github.com/discourse/discourse/pull/20139)